jetstream/cli.py

```diff
@@ -96,7 +96,9 @@ def run(project_id, dataset_id, date, experiment_slug, dry_run, config_file):
     # fetch experiments that are still active
     collection = ExperimentCollection.from_experimenter()
 
-    active_experiments = collection.end_on_or_after(date).of_type(("pref", "addon", "message"))
+    active_experiments = collection.end_on_or_after(date).of_type(
+        ("pref", "addon", "message", "nimbus")
```

Comment:
I'm guessing we want to analyse all Nimbus flavored experiments? I set their type as "nimbus" for now.